src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const bigImg = document.getElementById('largeImg');
+
+const images = document.querySelectorAll('.list-item__link');
+
+for (const image of images) {
+  image.addEventListener('click', (e) => {
+    e.preventDefault();
+
+    const place = image.getAttribute('href');
```

Comment:
Verify that the 'href' attribute of the image elements contains a valid image URL. Otherwise, setting the 'src' attribute of 'bigImg' may not work as expected.